graphql/mutation_resolver.go

```diff
@@ -149,18 +149,29 @@ func (r *mutationResolver) SetAnnotationMetadataLinks(ctx context.Context, taskI
 
 // SaveAdminSettings is the resolver for the saveAdminSettings field.
 func (r *mutationResolver) SaveAdminSettings(ctx context.Context, adminSettings restModel.APIAdminSettings) (*restModel.APIAdminSettings, error) {
-	oldSettings, err := evergreen.GetConfigWithoutSecrets(ctx)
+	oldSettings, err := evergreen.GetConfig(ctx)
```

Comment:
I'd want you to double check me on this but wouldn't this not matter even if it was withoutSecrets settings?